### PR TITLE
fix(console): warning for actions

### LINKED ISSUE #6317: Actions default and message
As an administrator I want to know the impact a failed action can have on my account, for example that I can get locked out. Per default we want to set the action to allow to fail, so I am still able to login. When I change to not allowed to fail and want to store the action we should show a message to the user, that this can have an impact.

**Acceptance Criteria**
- [ ] Per default the allowed to fail is true when saving an action
- [ ] I message is shown when the action will be stored and is set to not fail

**Additional Information**
Text to show:
Be aware that when the action is not allowed to fail all users of the organization will not be able to go pass the action, no matter if it is an administrator account or not. Example: When you trigger an action on authentication and it has an error (also a bug in the action), the user can't login anymore. We recommend having admin users in another organization or trying the scripts first in a dev environment or a dev organization.

### PR DESCRIPTION
**Pull request by peintnermax**
Closes #6317

### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

**Comment by peintnermax** [comment]
@hifabienne can you look at the texts it they are fine for you?.. I modified them to be clearer

File: console/src/assets/i18n/de.json [hunk 1]
```diff
@@ -742,6 +742,10 @@
     "TIMEOUT": "Timeout",
     "TIMEOUTINSEC": "Timeout in Sekunden",
     "ALLOWEDTOFAIL": "Scheitern erlaubt",
+    "ALLOWEDTOFAILWARN": {
+      "TITLE": "Warnung",
+      "DESCRIPTION": "Beachten Sie, dass durch das Deaktivieren dieser Einstellung der Vorgang zum Anmelden oder Anmelden von Benutzern der aktuellen Organisation unterbrochen werden kann und der Zugriff auf die Konsole zum Deaktivieren einer Aktion abgelehnt werden kann. Wir empfehlen, Administratorbenutzer in einer separaten Organisation einzurichten oder Skripte zunächst in einer Entwicklungsumgebung oder einer Entwicklungsorganisation auszuprobieren."
```
Comment (hifabienne) [inline console/src/assets/i18n/de.json:747]:
```suggestion
      "DESCRIPTION": "Beachten Sie, dass durch das Deaktivieren dieser Einstellung der Vorgang zum Anmelden von Benutzern der aktuellen Organisation unterbrochen werden kann und der Zugriff auf die Konsole zum Deaktivieren einer Aktion abgelehnt werden kann. Wir empfehlen, Administratorbenutzer in einer separaten Organisation einzurichten oder Skripte zunächst in einer Entwicklungsumgebung oder einer Entwicklungsorganisation auszuprobieren."
```

File: console/src/assets/i18n/en.json [hunk 2]
```diff
@@ -743,6 +743,10 @@
     "TIMEOUT": "Timeout",
     "TIMEOUTINSEC": "Timeout in seconds",
     "ALLOWEDTOFAIL": "Allowed To Fail",
+    "ALLOWEDTOFAILWARN": {
+      "TITLE": "Warning",
+      "DESCRIPTION": "Be aware that by disabling an action to fail, the process to log in or sign in users of the current organization can be interrupted and access to the console to disable an action can be declined. We recommend setting up admin users in a seperate organization or trying scripts in a dev environment or a dev organization first."
```
Comment (hifabienne) [inline console/src/assets/i18n/en.json:748]: Whats the difference between login and sign in? I think probably all of the text will have that duplicated now

Comment (peintnermax) [reply console/src/assets/i18n/en.json:748]: its the same.. I'll remove the second occurrence and change all other languages

Comment (peintnermax) [reply console/src/assets/i18n/en.json]: done

File: console/src/assets/i18n/de.json [hunk 3]
```diff
@@ -744,7 +744,7 @@
     "ALLOWEDTOFAIL": "Scheitern erlaubt",
     "ALLOWEDTOFAILWARN": {
       "TITLE": "Warnung",
-      "DESCRIPTION": "Beachten Sie, dass durch das Deaktivieren dieser Einstellung der Vorgang zum Anmelden oder Anmelden von Benutzern der aktuellen Organisation unterbrochen werden kann und der Zugriff auf die Konsole zum Deaktivieren einer Aktion abgelehnt werden kann. Wir empfehlen, Administratorbenutzer in einer separaten Organisation einzurichten oder Skripte zunächst in einer Entwicklungsumgebung oder einer Entwicklungsorganisation auszuprobieren."
+      "DESCRIPTION": "Beachten Sie, dass durch das Deaktivieren dieser Einstellung der Vorgang zum Anmelden von Benutzern der aktuellen Organisation unterbrochen werden kann und der Zugriff auf die Konsole zum Deaktivieren einer Aktion abgelehnt werden kann. Wir empfehlen, Administratorbenutzer in einer separaten Organisation einzurichten oder Skripte zunächst in einer Entwicklungsumgebung oder einer Entwicklungsorganisation auszuprobieren."
```
Comment (hifabienne) [inline console/src/assets/i18n/de.json:747]: First sentence feels a little hard to read, and just saw that it is in "per Sie"
What do you think about the following: 

```suggestion
      "DESCRIPTION": "Wenn du diese Einstellung deaktivierst, kann es dazu führen, dass sich Benutzer deiner Organisation nicht mehr anmelden können. Ausserdem kannst du dann nicht mehr auf die Konsole zugreifen, um die Action zu deaktivieren. Wir empfehlen, einen Administratorbenutzer in einer separaten Organisation zu erstellen oder Skripte zuerst in einer Entwicklungsumgebung oder einer Entwicklungsorganisation zu testen."
```

**Review by hifabienne — APPROVED** [review]

**Pull request merged**